docs/UserGuide.md

```diff
@@ -410,14 +413,35 @@ SweetRewards data are saved in the hard disk automatically after any command tha
 
 ### 4.17 Editing the data file
 
-SweetRewards data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+SweetRewards data are saved automatically as a [JSON](#6-glossary) file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
-⚠️ **Warning**: If your changes to the data file makes its format invalid, SweetRewards will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.<br>
+⚠️ <span style="color:orange">**Warning**:</span> If your changes to the data file makes its format invalid, SweetRewards will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.<br>
 Furthermore, certain edits can cause SweetRewards to behave in unexpected ways (e.g., if a value entered is outside of the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 
 --------------------------------------------------------------------------------------------------------------------
 
-# 5. Glossary
+# 5. Example Tutorial
```

Comment:
Perhaps you can say somewhere in the getting started or somewhere in the beginning that there is tutorial below. Hyperlink it to here if not those who don't scroll all the way down would not know there is a tutorial here